Frameworks/CoreText/CTLine.mm

```diff
@@ -322,8 +347,7 @@ CGRect CTLineGetImageBounds(CTLineRef line, CGContextRef context) {
  @Status Interoperable
 */
 double CTLineGetTypographicBounds(CTLineRef lineRef, CGFloat* ascent, CGFloat* descent, CGFloat* leading) {
-    _CTLine* line = static_cast<_CTLine*>(lineRef);
-
+    __CTLine* line = const_cast<__CTLine*>(lineRef);
```

Comment:
i'm not going to make logic changes in this CR.
That's out of scope, it should of been done initially to avoid modifying private vars.

---
In reply to: [112725004](https://github.com/Microsoft/WinObjC/pull/2586#discussion_r112725004) [](ancestors = 112725004)